types/peers.pp

```diff
@@ -0,0 +1,15 @@
+# @summary custom data type for an array with wireguard peers
+#
+# @author Tim Meusel <tim@bastelfreak.de>
+# @author Sebastian Rakel <sebastian@devunit.eu>
+#
+# @see https://www.freedesktop.org/software/systemd/man/systemd.netdev.html#%5BWireGuardPeer%5D%20Section%20Options
+
```

Comment:
This empty line means puppet-strings doesn't consider the comment above as documentation:
```suggestion
```